### ISSUE TITLE
Cannot add a policy to a virtual server

### ISSUE THREAD

**Issue by dtm80** [F5Networks/terraform-provider-bigip#861]
Hello,

I am trying to apply a bigip_ltm_policy to virtual server but I cant find the way to do it. In other issues lilke https://github.com/F5Networks/terraform-provider-bigip/issues/549  it is mentioned that is possible to attach it using an argument reference named "policies" inside the bigip_ltm_virtual_server https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server but this argument reference does not exist

**Comment by pgouband** [comment]
Hi @dtm80,

Thanks for the feedback, we will check the documentation.

Parameter "policies" in virtual server is the correct one to apply a ltm policy to a virtual server.
Here an example:

```terraform
resource "bigip_ltm_pool" "mypool" {
  name                = "/Common/test-pool"
  allow_nat           = "yes"
  allow_snat          = "yes"
  load_balancing_mode = "round-robin"
}
resource "bigip_ltm_policy" "testpolicy" {
  name     = "/Common/testpolicy"
  strategy = "first-match"
  requires = ["http"]
  controls = ["forwarding"]
  rule {
    name = "rule6"
    action {
      forward    = true
      connection = false
      pool       = bigip_ltm_pool.mypool.name
    }
  }
  depends_on = [bigip_ltm_pool.mypool]
}
resource "bigip_ltm_virtual_server" "http" {
  name        = "/Common/terraform_vs_http"
  destination = "10.12.12.12"
  port        = 80
  profiles    = ["/Common/http"]
  policies    = [bigip_ltm_policy.testpolicy.name]
  depends_on = [bigip_ltm_policy.testpolicy]
}
```

**Comment by dtm80** [comment]
I have tried the parameter policies in my lab and it does not work, we have used the lates version of terraform and 17 in BIGIP

So I could not find a way to attach the policy. As you can see in the screenshot wehn the resource is apply the attribute does not even exist

![image](https://github.com/F5Networks/terraform-provider-bigip/assets/144039032/5294b73f-5a09-453f-853e-ced1ba805242
![Capture](https://github.com/F5Networks/terraform-provider-bigip/assets/144039032/162c0b10-ad0b-40fe-a005-1adbd1ca66fa)

**Comment by pgouband** [comment]
Hi @dtm80,

I'm using latest version (1.19) and here the output of terraform plan, policies is listed.
Can you  share your main.tf?

```console
$ terraform plan -out test-policy
bigip_ltm_pool.mypool: Refreshing state... [id=/Common/test-pool]
bigip_ltm_policy.testpolicy: Refreshing state... [id=/Common/testpolicy]

Terraform used the selected providers to generate the following execution plan.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # bigip_ltm_virtual_server.http will be created
  + resource "bigip_ltm_virtual_server" "http" {
      + default_persistence_profile    = (known after apply)
      + destination                    = "10.12.12.12"
      + fallback_persistence_profile   = (known after apply)
      + firewall_enforced_policy       = (known after apply)
      + id                             = (known after apply)
      + ip_protocol                    = "tcp"
      + mask                           = (known after apply)
      + name                           = "/Common/terraform_vs_http"
      + per_flow_request_access_policy = (known after apply)
      + policies                       = [
          + "/Common/testpolicy",
        ]
      + port                           = 80
      + profiles                       = [
          + "/Common/http",
        ]
      + snatpool                       = (known after apply)
      + source                         = (known after apply)
      + source_address_translation     = (known after apply)
      + source_port                    = (known after apply)
      + state                          = "enabled"
      + trafficmatching_criteria       = (known after apply)
      + translate_address              = "enabled"
      + translate_port                 = "enabled"
      + vlans_enabled                  = false
    }

Plan: 1 to add, 0 to change, 0 to destroy.

───────────────────────────────────────────────────────────────────────────────

Saved the plan to: test-policy

To perform exactly these actions, run the following command to apply:
    terraform apply "test-policy"
```

**Comment by dtm80** [comment]
Hello, 

You are right, it works. So it is basically only a documentation issue that there is no argument reference for policies  inside of https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server

**Comment by RavinderReddyF5** [comment]
fixed in v1.20.0
